src/Components/GameCardList/GameCard/GameCard.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classes from './GameCard.module.scss';
+import fullstarIcon from '../../../images/star.png';
+import halfStarIcon from '../../../images/halfstar.png';
+import playerCountIcon from '../../../images/playercount.png';
+import playTimeIcon from '../../../images/time.png';
+
+const GameCard = (props) => {
+  const roundedRaiting = Math.round(props.game.average_user_rating * 2) / 2;
+  const ratingArr = [];
+
+  for (let i = 0; i < Math.floor(roundedRaiting); i++) {
+    ratingArr.push('full');
+  }
+  if (roundedRaiting % 1 > 0) {
+    ratingArr.push('half');
+  }
+
+  const raiting = ratingArr.map((star, index) => {
+    return star === 'full' ? (
+      <img
+        data-test="full-star"
+        src={fullstarIcon}
+        alt="one star"
+        key={index}
+      />
+    ) : (
+      <img
+        data-test="half-star"
+        src={halfStarIcon}
+        alt="half star"
+        key={index}
+      />
+    );
+  });
+
+  return (
+    <div data-test="component-game-card" className={classes.Container}>
```

Comment:
Same as above will not write it for all of them :) in regards to capital on the css name `Container`